src/test/java/io/vertx/ext/mongo/MongoClientTest.java

```diff
@@ -117,13 +139,15 @@ private void testFindBatch(BiFunction<CountDownLatch, ReadStream<JsonObject>, Li
     awaitLatch(latch);
     assertEquals(numDocs, foos.get().size());
     assertEquals("bar0", foos.get().get(0));
-    assertEquals("bar999", foos.get().get(numDocs - 1));
-
+    String bar = "bar";
+    for(int i=0;i<String.valueOf(numDocs).length()-1;i++){
+      bar = bar + "9";
+    }
+    assertEquals(bar, foos.get().get(numDocs - 1));
```

Comment:
Assert doesn't match the name of test?

Reply:
What do you mean?
We needed to change the "assertEquals" as we tried to keep your old assertEquals logic and also be able to use variable numDocs instead of the old fixed 3000 docs.